chicago/settings.py

```diff
@@ -123,14 +123,13 @@
                 "django.template.context_processors.request",
                 "django.contrib.auth.context_processors.auth",
                 "django.contrib.messages.context_processors.messages",
-                "councilmatic_core.views.city_context",
```

Comment:
this context allowed for a number of settings variables to be universally available in templates. we are adding them to each view `context` as needed